demo/src/app/components/timepicker/demos/seconds/timepicker-seconds.ts

```diff
@@ -9,6 +9,6 @@ export class NgbdTimepickerSeconds {
   seconds = true;
 
   toggleSeconds() {
-      this.seconds = !this.seconds;
+    this.seconds = !this.seconds;
```

Comment:
Is this clang-format, manual edit or sth else? Just wondering how this edit happened.